<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KnobDialogFrame</class>
 <widget class="QFrame" name="KnobDialogFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QRadioButton" name="decimalRadioButton">
       <property name="text">
        <string>raw</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="physicalRadioButton">
       <property name="text">
        <string>physically</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="configDirectCheckBox">
       <property name="text">
        <string>BB config direct</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="minValueLabel">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>min value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueInput" name="lineEditMinValue">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxValueLabel">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>max value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueInput" name="lineEditMaxValue">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="InheritLabel">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Inherit from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inheritComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QGroupBox" name="ScaleGroupBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Scale</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="ScaleMinAngleLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>min angle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ScaleMinAngleSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>°</string>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="ScaleMaxAngleLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>max angle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ScaleMaxAngleSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string>°</string>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QCheckBox" name="ScaleStepCheckBox">
            <property name="text">
             <string>step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ScaleStepDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>°</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>pos x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ScalePosXDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>pos y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ScalePosYDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QCheckBox" name="ScaleSizeCheckBox">
            <property name="text">
             <string>size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ScaleSizeDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QCheckBox" name="ScaleColorCheckBox">
            <property name="text">
             <string>user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ScaleColorPushButton">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="knobGroupBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Knob/Needle</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="needlePosXLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>pos x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="needlePosYLabel">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>pos y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="needleSizeCheckBox">
            <property name="text">
             <string>size</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="needleLineColorCheckBox">
            <property name="text">
             <string>line</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="needleLineColorPushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="needleFillColorCheckBox">
            <property name="text">
             <string>fill</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="needleFillColorPushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="needleSizeDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="needlePosXDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="needlePosYDoubleSpinBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QGroupBox" name="textGroupBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Value</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QCheckBox" name="textFontCheckBox">
            <property name="text">
             <string>user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="textFontPushButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>align</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="textAlignComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="nameGroupBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Name</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QCheckBox" name="nameFontCheckBox">
            <property name="text">
             <string>user</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nameFontPushButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>align</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="nameAlignComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="bitmapGroupBox">
     <property name="title">
      <string>Background Bitmap</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="bitmapLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="bitmapPushButton">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="fixedSizeRadioButton">
        <property name="text">
         <string>no scaling no fixed size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="resizeToFixedBitmapRadioButton">
        <property name="text">
         <string>resize window fixed to bitmap size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scaleToWindowRadioButton">
        <property name="text">
         <string>scale bitmap size to window size</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="IfReleaseFallbackToGroupBox">
     <property name="title">
      <string>If release fallback</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>to value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ValueInput" name="IfReleaseFallbackToLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="DragStatusSignalGroupBox">
     <property name="title">
      <string>Drag status</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DragStatusSignalLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueInput</class>
   <extends>QLineEdit</extends>
   <header>ValueInput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bitmapGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bitmapPushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ScaleMinAngleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ScaleMinAngleLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ScaleMaxAngleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ScaleMaxAngleLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>knobGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>needlePosYLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>knobGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>needlePosXLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitmapGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>bitmapLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitmapGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>fixedSizeRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitmapGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>resizeToFixedBitmapRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitmapGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>scaleToWindowRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
